components/cluster.py

```diff
@@ -99,26 +99,80 @@ def get_component(session_id):
         maxdate = maxdate + datetime.timedelta(days=1)
     items = []
     items.append(html.H1("Clustering"))
-    items.append(html.H3("Select a date range"))
+    items.append(html.H4("Compare clusters"))
+    tab1 = build_cluster_component(mindate, maxdate, session_id, "1")
+    tab2 = build_cluster_component(mindate, maxdate, session_id, "2")
     items.append(
-        html.Div([
-            date_picker.get_component(mindate, maxdate, None),
-            html.Div(
-                id='intermediate-value',
-                style={'display': 'none'},
-                children=[session_id]
-            ),
-            html.Div(id='output-container-date-picker-range')
+        dcc.Tabs([
+            dcc.Tab(label='Date 1', children=[tab1]),
+            dcc.Tab(label="Date 2", children=[tab2])
         ])
     )
     return html.Div(items)
 
 
+def build_cluster_component(mindate, maxdate, session_id, id="1"):
+
+    return html.Div([dcc.DatePickerRange(
+        id='date-pick-'+id,
+        min_date_allowed=mindate,
+        max_date_allowed=maxdate,
+    ),
```

Comment:
Here it would be better to reuse the `date_picker` component we have, you could just add an optional argument to its `get_component` function for passing a custom id